synapse/storage/databases/main/deviceinbox.py

```diff
@@ -743,15 +710,15 @@ def _remove_devices_from_device_inbox_txn(
 
         num_deleted, finished = await self.db_pool.runInteraction(
```

Comment:
num_deleted is now unused here - just have `_remove_dead_devices_from_device_inbox_txn` return the `finished` flag

Reply:
Ah yeah indeed, good point